department-of-veterans-affairs/abd-vro
# MCP 2444: Remove duplicate data model

## What was the problem?

Duplicate code presents maintainability issues. We need to copy over shared files in Dockerfile, according to Ryan.

Associated tickets or Slack threads:

- [MCP-2444](https://amida.atlassian.net/browse/MCP-2444)

## How does this fix it?[^1]

Add a line to copy over shared files to each python docker image.

## How to test this PR
- Step 1 Run end2end tests
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
